src/config.ts

```diff
@@ -53,7 +52,15 @@ export default class Config {
    * load the extends property, load the plugins and start the git remote interface.
    */
   async loadConfig(args: ArgsType) {
-    const explorer = cosmiconfig('auto');
+    const explorer = cosmiconfig('auto', {
+      searchPlaces: [
+        `.autorc`,
```

Comment:
could probably update the documentation to reflect what types of configs `auto` will load